### ISSUE TITLE
Support for "now" value in trial_end

### ISSUE THREAD

**Issue by jaime-fuhr** [stripe/stripe-dotnet#91]
The value 'now' can be passed into the trial_end field for a subscription which ends the customer's trial immediately.   See https://stripe.com/docs/api#update_subscription.

I didn't see a way to do this in the latest build, so I modified the 1.5.4 source we are using, changing TrialEndInternal in StripeCustomerUpdateSubscriptionOptions to:

[JsonProperty("trial_end")]
        internal string TrialEndInternal
        {
            get
            {
                if (!TrialEnd.HasValue) {
                    return null;
                }

```
            if (TrialEnd.Value == new DateTime(1900, 1, 1)) {
                return "now";
            } else {

                var diff = TrialEnd.Value - new DateTime(1970, 1, 1);

                return Math.Floor(diff.TotalSeconds).ToString();
            }
        }
    }
```

Basically changing the return result to string and checking TrialEnd for a value of 1/1/1900.  If so, return "now".

Not sure if this is the best way to implement, but I thought I would pass this along.

**Comment by Thwaitesy** [comment]
This solution is awesome when creating a subscription on its own, however, if I'm creating a customer, adding the subscription, token etc - id love to be able to set the trial period to end 'now' there also.

Currently im using the hack of ".TrialEnd = DateTime.UtcNow.AddSeconds(1)"

Tempted to create a customer then create a subscription after that but would be nice to have this all in the one API call.
